### PR TITLE
Workspace trust - add --disable-workspace-trust flag

### PR DESCRIPTION
**Pull request by lszomoru**

### CODE REVIEW

**Comment by lszomoru** [comment]
@bpasero, could you please review the files specific to the `EnvironmentService` 
@sbatten, could you please review the files specific to the `WorkspaceTrustService`

**Review by bpasero — APPROVED** [review]
LGTM, web always has workspace trust disabled? Is that intentional?

**Comment by lszomoru** [comment]
@bpasero, yes. Workspace trust is intentionally turned off in web.

File: src/vs/platform/workspace/common/workspaceTrust.ts [hunk 1]
```diff
@@ -41,9 +41,10 @@ export interface IWorkspaceTrustManagementService {
 	onDidChangeTrustedFolders: Event<void>;
 
 	readonly workspaceResolved: Promise<void>;
+	readonly workspaceTrustEnabled: boolean;
 	readonly workspaceTrustInitialized: Promise<void>;
```
Comment (sbatten) [inline src/vs/platform/workspace/common/workspaceTrust.ts:45]: silly thing, but this should be above the promises

**Review by sbatten — APPROVED** [review]

File: src/vs/platform/workspace/common/workspaceTrust.ts (hunk 1 above)
Comment (lszomoru) [reply src/vs/platform/workspace/common/workspaceTrust.ts:45]: Done

**Pull request merged**